### DISCUSSION TITLE
Imputing category without orthology

### DISCUSSION THREAD

**Discussion by llrs** [geneontology/helpdesk#502] [Q&A]
I am looking for software that could annotate genes to specific ontologies (those related to immune system and microbiota).

I've seen that [most IEA comes from orhtologies](https://github.com/geneontology/helpdesk/discussions/369) are there some tools to infer/impute annotation without using ornthology data?
Ideally I'd like to have some gene expression data (it is more abundant and accessible) to infer GO categories for the genes. 

Sorry if I missed something, I haven't found publications related to this and I don't know if this is because is not feasible or that it is not an area of active research.

**Comment by suzialeksander** [comment]
Hi @llrs 

Although orthology is often used for making high quality automated annotations, the amount of inferred annotations varies between organisms. What organism(s) are you investigating? We can better help you if we have a little more information about what you're trying to do.

If you are looking to group existing annotations into branches of the ontology related to immune system, etc., we can help you with tools like [GO Term Mapper](https://go.princeton.edu/cgi-bin/GOTermMapper), which will organize genes into "bins" for specified GO terms.

Our recommendation for annotating novel genomes is to [submit transcripts to NCBI](https://www.ncbi.nlm.nih.gov/). These submissions will reach [UniProt](https://www.uniprot.org/), where [InterPro2GO](https://www.ebi.ac.uk/GOA/InterPro2GO) will automatically create high-quality ([IEA](https://geneontology.org/docs/guide-go-evidence-codes/)) GO annotations. These annotations will be made available in a future GO release. GO does not recommend that groups create their own IEAs with internal tools due to reproducibility and accuracy concerns.



Research groups who wish to generate manual GO annotations from experimental data- for instance, new annotations for genes of a species that is not currently annotated in GO, or annotating specific genes for a given biological research domain- can contribute these annotations to the GO knowledgebase.

**Reply by llrs** [reply]
Hi @suzialeksander thanks for the quick anser. I want to find new potential immune receptors on humans using computational tools (I don't have experimental data). 
[InterPro2GO](https://www.ebi.ac.uk/GOA/InterPro2GO) was " evaluated at 91-100% accurate(Camon et. al . 2005)." It's been 14 years since the last evaluation. Aren't there more recent evaluations or better methods since then?
